### DISCUSSION TITLE
Airports in ND both sides

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4091] [General]
![ND](https://user-images.githubusercontent.com/76531893/112132103-87077680-8bca-11eb-9362-e5a475b03852.jpg)

Hi simmers and developers,

I figured out that the ND of cpt and 1O shows different airports? Is this a known issue?

Greets Klaus

**Comment by CptBayer** [comment]
![ND 2](https://user-images.githubusercontent.com/76531893/112132741-36dce400-8bcb-11eb-82e4-eed683b04d08.jpg)

... and it took round about 90 seconds till the rest of the airports are displayed :-p .... 

greets Klaus

**Comment by hotshotp** [comment]
You have them zoomed out at different ranges on the NDs

**Comment by tracernz** [comment]
fs9gps API is very slow to start returning initial results, hence it takes a while after you first select.
